rules/contrib/views.py

```diff
@@ -111,10 +109,10 @@ def post_update(request, post_id):
     used.
     """
     def decorator(view_func):
-        @wraps(view_func, assigned=available_attrs(view_func))
+        @wraps(view_func, assigned=WRAPPER_ASSIGNMENTS(view_func))
```

Comment:
You must use it as `@wraps(view_func)` — WRAPPER_ASSIGNMENTS is not callable, and it is the default value for “assigned” so it can be omitted.

Reply:
Thank you, done!